src/main/java/it/gov/pagopa/pu/classification/repository/AssessmentsRepository.java

```diff
@@ -8,4 +8,6 @@
 @RepositoryRestResource(path = "assessments")
 public interface AssessmentsRepository extends JpaRepository<Assessments,String> {
 
+  Assessments getByOrganizationIdAndDebtPositionTypeOrgCodeAndAssessmentName(Long organizationId, String debtPositionTypeOrgCode, String assessmentName);
```

Comment:
There is not UNIQUE constraint on these fields!
@marcomatteuccieng should we introduce a semantic ID on this entity?